bindata/v3.11.0/kube-controller-manager/clusterrolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: openshift-kube-controller-manager-cluster-admin
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
the kubeconfig for the "other" daemonset was getting a `system:masters` identity, so this is equivalent for now.

Reply:
Should we create an issue that this is to be fixed?

Reply:
> Should we create an issue that this is to be fixed?

Sure, but I wouldn't start working on it now